people/alon-lavie.md

```diff
@@ -0,0 +1,24 @@
+---
+description: Machine translation researcher and co-developer of METEOR
+---
+# Alon Lavie
+Alon Lavie is a machine translation researcher. He received Ph.D. in computer science at Carnegie Mellon University in 1995. 
+
+His main research interests are Machine Translation adaptation approaches, automatic metrics for machine translation 
+evaluation, quality estimation, and methods for multi-engine machine translation system combination.
+
+Lavie served as president at the International Association for Machine Translation and the Association for Machine 
+Translation in the Americas. He was also a member of the Association for Computational Linguistics (ACL).  
+
+In 1996-2015, Lavie worked as a research professor at the Language Technologies Institute at Carnegie Mellon University.
+
+In 2004, Lavie together with Michael J. Denkowski developed and released [METEOR](/metric/meteor.md), an automatic metric 
+for the evaluation of machine translation output. 
+
+In 2009, he co-founded Safaba Translation Solutions that offered automated translation solutions to global enterprises. 
+In 2015, it was aquired by Amazon. 
+
+In 2015-2019, Lavie was a senior manager at Amazon, where he led and managed the Amazon Machine Translation R&D group in Pittsburgh. 
+
+In 2019, he moved to Unbabel as a vice president of Language Technologies in charge of leading and managing US AL lab based in Pittsburgh, 
```

Comment:
Vice President is a title, so titlecase

Reply:
> in charge of leading and managing

3 ways of saying the same thing ;-)

> to lead the AI lab.

but I don't think it's totally necessary.